dali/c_api/c_api_test.c

```diff
@@ -12,15 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef DALI_OPERATORS_OPERATORS_H_
-#define DALI_OPERATORS_OPERATORS_H_
-
-namespace dali {
-/**
- * @brief The function to reference, when one needs to make sure DALI operators shared
- *        object is actually linked against.
- */
-DLL_PUBLIC void InitOperatorsLib();
-}  // namespace dali
-
-#endif  // DALI_OPERATORS_OPERATORS_H_
+#include "dali/c_api.h"
```

Comment:
what is this file for?

Reply:
Building `c_api.h` with C compiler - checking if there're no build errors at the very least